creator-onboarding/presents.md

```diff
@@ -14,10 +15,7 @@ Before proceeding to mainnet upload, please verify that your script behaves as e
 2. Ensure your wallet is funded with enough ETH to pay the gas fees for the upload and unpausing process. For more information on the estimated cost of these steps, [click here.](readme/readme.md#cost)
 3. Submit one field at a time. Wait for that transaction to clear before attempting the next field.
 4. When you upload your script, please copy it exactly from testnet.
-5. Include the following in the Project Base URI field: `https://api.artblocks.io/token/`
-   * **Important:** If you are setting this field for an Engine project, rather than an Art Blocks project directly, the `baseTokenURI` field should follow a slightly different structure. Please see the [Art Blocks Engine documentation](../art-blocks-engine-onboarding/art-blocks-engine-101/adding-new-project-shells.md) for more info.
```

Comment:
I removed this because I don't think it is required for V3 core contracts, and this shouldn't have to be supporting V2 contracts at this time IIUC.